AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6511 @@
+{
```

Comment:
<p>Nice. Get rid of the y labels</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>